quick-start/src/main/java/com/marklogic/quickstart/service/MappingManagerService.java

```diff
@@ -66,10 +69,6 @@ public MappingModel createMapping(String projectDir, MappingModel newMapping) th
         scaffolding.createMappingDir(newMapping.getName());
         Mapping mapping = mappingManager.createMappingFromJSON(newMapping.toJson());
         mappingManager.saveMapping(mapping);
-        Path dir = envConfig().getMlSettings().getHubMappingsDir().resolve(newMapping.getName());
-        if (dir.toFile().exists()) {
-            watcherService.watch(dir.toString());
-        }
         return getMapping(mapping.getName());
     }
```

Comment:
Should this piece of code be removed ? After a mapping is created, we do have to install user modules right ? Also deleteMapping() still unwatches the mapping dir when called

Reply:
Yes, it adds another watch that interferes with the original watch laid over the plugin directory, which causes a race condition from time to time.

And as delete mapping removes a directory, so to be safe we force-remove any watch that maybe placed on it as a precautionary measure.